spec/README.md

```diff
@@ -66,3 +67,31 @@ A **_term_** looks like this when it is defined in this specification.
 A reference to a _term_ looks like this.
 
 > Examples are non-normative and styled like this.
+
+### Stability Policy
+
+Updates to this specification MUST NOT change
+the syntactical meaning, the formatting, or other behaviour
```

Comment:
When I first read this, for some reason I thought "formatting referred to the layout of the code, not the runtime behavior. Perhaps be specific here?

```suggestion
the syntactical meaning, the runtime output, or other behaviour
```